### DISCUSSION TITLE
Swell-Contentful integration

### DISCUSSION THREAD

**Discussion by emilsbee** [swellstores/community#407] [Help] [answered]
Hi everyone,

I see that there is a Contentful integration for Swell which seems to work fine, however it's missing one pretty crucial feature and that is referencing Swell categories in Contentful. Is it on the roadmap for Swell to implement this feature? Or perhaps there's a workaround for this?

Thanks!

**Comment by ericingram** [comment, marked as answer]
We don't have it on a roadmap, but w will likely migrate all the existing integrations to Apps which would be open source. In the meantime you might consider joining  our Apps alpha program and see if you can make it work the way you want (and we can provide some guidance).
